assets/app/styles/_core.less

```diff
@@ -301,6 +314,11 @@ html, body {
   }
 }
 
+.required:before {
+  content: '* ';
```

Comment:
That's what I had before, but I took it out experimenting with spans.... :-/

I'll fix both issues.